pkg/kubectl/cmd/proxy.go

```diff
@@ -83,14 +83,33 @@ func NewCmdProxy(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("address", "", "127.0.0.1", "The IP address on which to serve on.")
 	cmd.Flags().Bool("disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	cmd.Flags().StringP("unix-socket", "u", "", "Unix socket on which to run the proxy.")
+	cmd.Flags().StringP("curl", "", "", "Raw URI to request from the proxy server.")
 	return cmd
 }
 
+func curl(f *cmdutil.Factory, uri, address string, port int) error {
+	restClient, err := f.RESTClientProxy(address, port)
+	if err != nil {
+		return err
+	}
+
+	return cmdutil.PrintRESTClientStream(restClient, uri)
+}
+
 func RunProxy(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
 	path := cmdutil.GetFlagString(cmd, "unix-socket")
 	port := cmdutil.GetFlagInt(cmd, "port")
 	address := cmdutil.GetFlagString(cmd, "address")
 
+	uri := cmdutil.GetFlagString(cmd, "curl")
+
+	if uri != "" {
```

Comment:
What's the difference? Also, that would be inconsistent with the rest of this file, there are some previously existing `if something == ""` statements.